packages/oui-datagrid/src/datagrid.controller.js

```diff
@@ -321,18 +324,20 @@ export default class DatagridController {
     }
 
     getSelectedRows () {
-        return this.selectedRows.reduce((result, isSelected, index) => {
-            if (isSelected) {
-                result.push(this.displayedRows[index]);
-            }
-            return result;
-        }, []);
+        return this.selectedRows;
     }
 
     toggleRowSelection (index, isSelected) {
+        const row = this.displayedRows[index];
```

Comment:
Use either `displayed` or `current` for all variables related to the things on the current page :)

Reply:
(I like `displayed` better, `current` doesn't really make sense)